api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/TaskTranslatorImpl.java

```diff
@@ -0,0 +1,44 @@
+package org.openmrs.module.fhir2.api.translators.impl;
+
+import lombok.AccessLevel;
+import lombok.Setter;
+import org.hl7.fhir.r4.model.Enumeration;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Task;
+import org.openmrs.module.fhir2.api.translators.TaskTranslator;
+import org.springframework.stereotype.Component;
+
+@Component
+@Setter(AccessLevel.PACKAGE)
+public class TaskTranslatorImpl implements TaskTranslator {
+	
+	@Override
+	public Task toFhirResource(org.openmrs.module.fhir2.Task openmrsTask) {
+		Enumeration<Task.TaskStatus> fhirStatus = new Enumeration(new Task.TaskStatusEnumFactory());
+		Enumeration<Task.TaskIntent> fhirIntent = new Enumeration(new Task.TaskIntentEnumFactory());
+		
+		fhirStatus.setValue(openmrsTask.getStatus());
```

Comment:
These next few lines don't compile. I think the implementation for this method should be something like this:

```java
Task task = new Task();

if (openmrsTask != null) {
	task.setId(openmrsTask.getUuid());
	task.setStatus(Task.TaskStatus.valueOf(openmrsTask.getStatus().name()));
	task.setIntent(Task.TaskIntent.valueOf(openmrsTask.getIntent().name()));
}

return task;
```